src/app/components/Input/Input.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './Input.module.css';
+
+function Input(): JSX.Element {
```

Comment:
This component has a very generic name. An input could be a password input, a text field etc. 
But this is a specific input with a fix label and placeholder.
I would rename this to e.g. `HabitInput` or add properties for the label and placeholder, to support different kind of inputs.
In addition, you will need a `value` and `onChange` to control the input. This could be done later too.